### ISSUE TITLE
babel/polyfill not found

### ISSUE THREAD

**Issue by h1mal** [csscomb/csscomb.js#473]
After calling csscomb, there is an error message.
```
$ csscomb
module.js:442
    throw err;
    ^

Error: Cannot find module 'babel/polyfill'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/csscomb/node_modules/csscomb-core/lib/core.js:6:3)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)

```

**Comment by skill83** [comment]
same here

**Comment by hermanbergwerf** [comment]
Yeah me too

**Comment by hermanbergwerf** [comment]
*Use `sudo npm install csscomb@3.1.7 -g` to get the current stable version

**Comment by tonyganch** [comment]
Thank you for reports, guys.
Can you specify the version of csscomb you're using please?

**Comment by h1mal** [comment]
Error occurs in 4.0.0-alpha.
No problems in 3.1.7/3.1.8

**Comment by skill83** [comment]
Error occurs in 3.9.5

**Comment by birdspider** [comment]
@skill83 , there is no `csscomb-3.9.5` published on npm ? are you talking about `npm`'s version ? (which is mostly irrelevant here)

**Comment by mkhazov** [comment]
`Error: Cannot find module 'babel/polyfill'`
You can see it this in online test as well: https://tonicdev.com/npm/csscomb-core

**Comment by timoxley** [comment]
@tonyganch: The current published version of `csscomb` (`4.0.0-alpha`) is broken due to a missing dependency in `csscomb-core`. Can you please `npm deprecate` these and `npm publish` new, fixed versions?

**Comment by timoxley** [comment]
This is primarily a problem because the alpha build is tagged as `latest` on npm; *every* user who runs `npm install csscomb` is going to get this broken version. Development builds should not be tagged with `latest`.

**Comment by tonyganch** [comment]
Should be fixed now, please run `npm update` and let me know if the problem remains?

**Comment by tonyganch** [comment]
@timoxley, thank you very much for the hint about dist tag. I've removed `latest` from `4.0.0-alpha` and pinned it to `3.1.7`

**Comment by timoxley** [comment]
@tonyganch works! thanks. Note you could add 4.0.0-alpha under an "alpha" tag so someone could go `npm install csscomb@alpha` and it would always pull down whatever the latest alpha is.

**Comment by dperetti** [comment]
Still bites me:

```
➜  ~  npm install csscomb -g
npm WARN deprecated node-uuid@1.4.0: use uuid module instead
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
/usr/local/bin/csscomb -> /usr/local/lib/node_modules/csscomb/bin/csscomb
/usr/local/lib
└─┬ csscomb@4.0.0
  ├─┬ glob@7.1.1
  │ ├── fs.realpath@1.0.0
  │ ├─┬ inflight@1.0.6
  │ │ └── wrappy@1.0.2
  │ ├── inherits@2.0.3
  │ ├── once@1.4.0
  │ └── path-is-absolute@1.0.1
  ├── gonzales-pe@3.4.7
  ├─┬ minimatch@3.0.2
  │ └─┬ brace-expansion@1.1.6
  │   ├── balanced-match@0.4.2
  │   └── concat-map@0.0.1
  ├── minimist@1.1.3
  ├── vow@0.4.4
  └─┬ vow-fs@0.3.2
    ├─┬ glob@3.2.8
    │ └─┬ minimatch@0.2.14
    │   ├── lru-cache@2.7.3
    │   └── sigmund@1.0.1
    ├── node-uuid@1.4.0
    └── vow-queue@0.3.1

➜  ~  csscomb
module.js:472
    throw err;
    ^

Error: Cannot find module 'babel-polyfill'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/csscomb/lib/core.js:4:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)

```

**Comment by h1mal** [comment]
Me, too. I downgraded to 3.1.7. This one still works.
sudo npm uninstall -g csscomb
sudo npm install -g csscomb@3.1.7

**Comment by tonyganch** [comment]
Thank you a lot for report!
The issue should be fixed with v4.0.1, please let me know if it's still there.

**Comment by h1mal** [comment]
v4.0.1 works perfectly!
Thank you very much!
